### DISCUSSION TITLE
Google Translate

### DISCUSSION THREAD

**Discussion by GenShibe** [catppuccin/catppuccin#2515] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the suggested port be themed?

userstyles

### Any additional comments?

![image](https://github.com/user-attachments/assets/3f834b9a-1124-4c68-a433-77995207c67d)
made a lil proof of concept

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by ignamartinoli** [comment]
It looks really nice. It's a pity that it didn't catch attention

**Reply by WalkQuackBack** [reply]
> It looks really nice. It's a pity that it didn't catch attention

The way Google Translate is structured (As of September 20th) makes ports like this extremely difficult to maintain, especially when big renovations are happening and multiple complete hash rerolls can happen within a single week. Maybe one day Google Translate will be migrated to the newer system...
